exercises/practice/roman-numerals/.docs/instructions.md

```diff
@@ -20,8 +20,7 @@ into stone tablets).
  7  => VII
 ```
 
-There is no need to be able to convert numbers larger than about 3000.
-(The Romans themselves didn't tend to go any higher)
+The largest number that can be represented in this notation is 3,999.
```

Comment:
Like @junedev  mentioned previously, changes to this file must be discussed first in the problem specifications repo, so we won't make them here.